pkg/docker/registry/auth.go

```diff
@@ -11,8 +11,8 @@ import (
 
 	"github.com/containers/image/v5/pkg/docker/config"
 	"github.com/containers/image/v5/types"
-	"github.com/distribution/distribution/v3/registry/client/auth/challenge"
```

Comment:
this package is now internal

Reply:
what's the reason behind needing to make this one internal?

Reply:
github.com/distribution/distribution has made it internal so we can no longer import it

https://github.com/distribution/distribution/tree/bc6e81e1b9a8017a66fc56a55913b92930941feb/internal/client/auth/challenge